bangazonapi/models/product.py

```diff
@@ -61,8 +61,9 @@ def average_rating(self):
         total_rating = 0
         for rating in ratings:
             total_rating += rating.rating
-
-        avg = total_rating / len(ratings)
+        try: avg = total_rating / len(ratings)
+        except ZeroDivisionError:
+            avg = total_rating
```

Comment:
The lowest possible rating is actually 0.  Is there a way to return something here that would differentiate it from a situation where something has an actual average of 0?